tests/utils/constants/config.go

```diff
@@ -119,3 +119,12 @@ func GetKubeConfigDir() string {
 	}
 	return configDir
 }
+
+func GetOutputManifestsDir() string {
```

Comment:
What is this used for?

Reply:
This is a leftover of my changes ... Removed